pkg/cmd/issue/view/fixtures/issueView_previewFullComments.json

```diff
@@ -310,9 +310,68 @@
                   }
                 }
               ]
+            },
+            {
+              "author": {
+                "login": "g14a"
+              },
+              "authorAssociation": "CONTRIBUTOR",
+              "body": "@vilmibm @ampinsk  I tried solving this. Issues and PRs look like the following respectively. Does it look okay or can it be better from a design perspective?\n## Issues: \n<img width=\"854\" alt=\"image\" src=\"https://user-images.githubusercontent.com/17702388/113286902-6ea20500-930a-11eb-8860-754107c67b22.png\">\n## PRs:\n<img width=\"854\" alt=\"image\" src=\"https://user-images.githubusercontent.com/17702388/113286974-8c6f6a00-930a-11eb-98da-41e159056438.png\">\nDo you want to take a look at the PR which fixes this?",
```

Comment:
Since you've only edited the `Render` function and nothing else in the codebase, let's make a test that exercises only the Render function and not `issue view`. This test has a lot of setup just to exercise one tiny piece of changed functionality.